firmware/quadruna/FSM/src/tasks.c

```diff
@@ -158,6 +160,10 @@ void tasks_preInit(void)
     LOG_INFO("FSM reset!");
 }
 
+static const FsmShdnConfig fsm_shdn_pin_config = { 1, fsm_shdn };
+
+static BoardShdnNode fsmBshdnNodes[1] = { { &io_get_FSM_SHDN_OK, &app_canTx_FSM_FSM_SHDN_OK_Status_set } };
```

Comment:
nitpick: 
variables should be `snake_case`, so `fsm_bspd_nodes`. Also, should this be called `fsm_shdn_nodes`?

Reply:
Here and elsewhere.